flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BaseWriter.java

```diff
@@ -116,6 +121,11 @@ public void flush(boolean endOfInput) {
         }
         logger.info("Validating all pending append responses in subtask {}", subtaskId);
         validateAppendResponses(true);
+        // .flush() is called at checkpoint, resetting the counters after all tasks are done.
```

Comment:
Instead of `.flush()`, just `Writer's flush` is more readable.

Reply:
Fixed.